modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/edit_object_relationship.jsp

```diff
@@ -17,130 +17,25 @@
-						<aui:option label="prevent" selected="<%= Objects.equals(objectRelationship.getDeletionType(), ObjectRelationshipConstants.DELETION_TYPE_PREVENT) %>" value="<%= ObjectRelationshipConstants.DELETION_TYPE_PREVENT %>" />
-					</aui:select>
-				</div>
-			</div>
-
-			<div class="side-panel-content__footer">
-				<aui:button cssClass="btn-cancel mr-1" name="cancel" value='<%= LanguageUtil.get(request, "cancel") %>' />
-
-				<aui:button disabled="<%= !objectDefinitionsRelationshipsDisplayContext.hasUpdateObjectDefinitionPermission() || objectRelationship.isReverse() %>" name="save" type="submit" value='<%= LanguageUtil.get(request, "save") %>' />
-			</div>
-		</div>
-	</form>
-</liferay-frontend:side-panel-content>
-
-<script>
-	function <portlet:namespace />saveObjectRelationship() {
-		const localizedInputs = document.querySelectorAll(
-			"input[id^='<portlet:namespace />'][type='hidden']"
-		);
-
-		const deletionType = document.getElementById(
-			'<portlet:namespace />deletionType'
-		);
-
-		const localizedLabels = Array(...localizedInputs).reduce(
-			(prev, cur, index) => {
-				if (cur.value) {
-					prev[cur.id.replace('<portlet:namespace />label_', '')] =
-						cur.value;
-				}
-
-				return prev;
-			},
-			{}
-		);
-
-		Liferay.Util.fetch(
-			'/o/object-admin/v1.0/object-relationships/<%= objectRelationship.getObjectRelationshipId() %>',
-			{
-				body: JSON.stringify({
-					deletionType: deletionType.value,
-					label: localizedLabels,
-				}),
-				headers: new Headers({
-					'Accept': 'application/json',
-					'Content-Type': 'application/json',
-				}),
-				method: 'PUT',
-			}
-		)
-			.then((response) => {
-				if (response.status === 401) {
-					window.location.reload();
-				}
-				else if (response.ok) {
-					Liferay.Util.openToast({
-						message:
-							'<%= LanguageUtil.get(request, "the-object-relationship-was-updated-successfully") %>',
-						type: 'success',
-					});
-
-					setTimeout(() => {
-						const parentWindow = Liferay.Util.getOpener();
-						parentWindow.Liferay.fire('close-side-panel');
-					}, 1500);
-				}
-				else {
-					return response.json();
-				}
-			})
-			.then((response) => {
-				if (response && response.title) {
-					Liferay.Util.openToast({
-						message: response.title,
-						type: 'danger',
-					});
-				}
-			});
-	}
-</script>
\ No newline at end of file
+	<react:component
+		module="js/components/EditObjectRelationship"
+		props='<%=
+			HashMapBuilder.<String, Object>put(
+				"hasUpdateObjectDefinitionPermission", objectDefinitionsRelationshipsDisplayContext.hasUpdateObjectDefinitionPermission()
+			).put(
+				"isReverse", objectRelationship.isReverse()
+			).put(
+				"objectRelationship", objectDefinitionsRelationshipsDisplayContext.getObjectRelationshipJSONObject(objectRelationship)
+			).put(
+				"objectRelationshipId", objectRelationship.getObjectRelationshipId()
```

Comment:
Is this variable "objectRelationshipId" the same of the one that "objectRelationship" contains ?(objectDefinitionsRelationshipsDisplayContext.getObjectRelationshipJSONObject(objectRelationship))